### PR TITLE
Remove deprecated objects

### LINKED ISSUE #22000: Remove deprecated objects
## Reason
Another stab at removing some deprecated objects that have been replaced by other functionality.

## Design
Remove files, fix fallout.

## Impact
Maintenance

### PR DESCRIPTION
**Pull request by dschwen**
Closes #22000

### CODE REVIEW

File: modules/stochastic_tools/src/samplers/MonteCarloSampler.C [hunk 1]
```diff
@@ -11,10 +11,6 @@
 #include "Distribution.h"
 
 registerMooseObjectAliased("StochasticToolsApp", MonteCarloSampler, "MonteCarlo");
```
Comment (GiudGiud) [inline modules/stochastic_tools/src/samplers/MonteCarloSampler.C:13]: this isnt recognized by the docs it seems

**Review by GiudGiud — CHANGES_REQUESTED** [review]
MASTODON and moltres still have NodalMax
some docs issue, I d recommend just removing the aliased registering or doubling it up with a regular registering

File: modules/stochastic_tools/src/samplers/MonteCarloSampler.C (hunk 1 above)
Comment (dschwen) [reply modules/stochastic_tools/src/samplers/MonteCarloSampler.C:13]: This seems to be used all over the place though...

Comment (dschwen) [reply modules/stochastic_tools/src/samplers/MonteCarloSampler.C:13]: fixed it

**Comment by dschwen** [comment]
> MASTODON and moltres still have NodalMax some docs issue, I d recommend just removing the aliased registering or doubling it up with a regular registering

PRs for both are up. Moltres needs a developer to activate testing though.

**Comment by GiudGiud** [comment]
@lindsayad you still involved in moltres?

**Comment by lindsayad** [comment]
I believe I still have merge rights…

**Pull request closed without merging**